### PR TITLE
docs: Update appliance build documentation

### LINKED ISSUE #841: [DOC] Appliance build documentation has errors
Whilst testing #840 I found a few small issues with the documentation for building the appliance.

I also suggest a small improvement to the `build.sh` script to avoid having to commit local changes to the `photon-build.json` file required to build the appliance.

I will file a PR for the above.

### PR DESCRIPTION
**Pull request by dsbibby**
## Summary

Please describe your changes and what you plan to introduce as part of this pull request submission. If it fixes a bug or resolves a feature request, be sure to link to the issue numbers following the template below.

Small documentation updates to address recent changes to the size of the appliance.
Small improvement suggested to the `build.sh` script.

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [x] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [x] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [x] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commits related to your change
- [x] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [x] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [x] Documentation changes
- [ ] Other (please describe):

## Resolved Issues

List of Issues closed or resolved by this PR. Add multiple `Closes` keyword followed by the issue number (e.g. Closes #ISSUE-NUMBER)

Closes #841 

## Testing Verification

* Built appliance and tested as part of #840 

## Additional Information

* Any other details you wish to include or mention

If you have any questions/comments, feel free to reach out to team on Slack [#vcenter-event-broker-appliance](https://vmwarecode.slack.com/archives/CQLT9B5AA)

Thank you from the VEBA Team! 🥳

### CODE REVIEW

**Comment by dsbibby** [comment]
Hopefully I've fixed the mess I made of this PR initially!

File: docs/kb/contribute-appliance.md [hunk 1]
```diff
@@ -23,6 +24,7 @@ cta:
   * [Packer](https://www.packer.io/intro/getting-started/install.html){:target="_blank"} (v1.6.3 or greater)
   * [jq](https://stedolan.github.io/jq/){:target="_blank"}
 * Development machine must have the firewall disabled for the duration of the build
+> **Note:** It has been seen that Packer can bind to an IPv6 on the development machine - you maywish to disable IPv6!
```
Comment (embano1) [inline docs/kb/contribute-appliance.md:27]:
```suggestion
> **Note:** It has been seen that Packer can bind to an IPv6 on the development machine - you may wish to disable IPv6!
```

File: build.sh [hunk 2]
```diff
@@ -16,7 +16,7 @@ if ! hash jq 2>/dev/null; then
     exit 1
 fi
 
-if [[ ! -z $(git status -s) ]]; then
+if [[ ! -z $(git status -s | grep -vE 'photon-builder.json') ]]; then
```
Comment (embano1) [inline build.sh:19]: question: is this a required file or can we put this in a `.gitignore`?

**Comment by embano1** [comment]
Thx sir! Can you please update your commit title/body with:

- `fix: ` prefix in the title
- `Closes: #841` in the body?

So it renders nicely in the release notes ;)

File: build.sh (hunk 2 above)
Comment (dsbibby) [reply build.sh]: It's required to be in the repo I think, but needs modifying for each developer's own environment. Currently those changes have to be committed in order to build the appliance.

**Review by lamw — APPROVED** [review]
LGTM

**Pull request merged**